services/quiz.js

```diff
@@ -1,19 +1,41 @@
 const conjugate = require('./conjugation');
+const utils = require('./spanishUtils');
```

Comment:
@sdee files should use consistent naming convention. In javascript it is hyphenated filenames. So this should be spanish-utils.js